packages/core/admin/admin/src/content-manager/components/DynamicTable/CellContent/utils/hasContent.js

```diff
@@ -14,7 +15,13 @@ export default function hasContent(type, content, metadatas, fieldSchema) {
       return content.length > 0;
     }
 
-    return !isEmpty(content[mainFieldName]);
+    const value = content[mainFieldName];
+
+    if (isFieldTypeNumber(mainFieldType) && mainFieldName !== 'id') {
+      return !Number.isNaN(value);
+    }
```

Comment:
@soupette If an ID field is set as main field of a component, it reports itself to be an integer. There is no attribute, which allows me to find out if it is an ID field (I think). Do you have any other idea on how to get this information, or are we ok with hard-coding ID fields?

Turns out: `_.isEmpty(number)` returns false. 🤦🏼

Reply:
We can hardcode `id`, `createdAt`, `updatedAt`, `publishedAt` as they are known fields.